CHANGELOG.md

```diff
@@ -13,6 +14,44 @@ The below sections record the changes for each API version in each release as fo
 * for subsequent release-candidate(s), only the delta to the previous release-candidate
 * for a public release, the consolidated changes since the previous public release
 
+# r1.2
+
+## Release Notes
+
+This public release contains the definition and documentation of
+* qos-booking v0.1.0
+* qos-booking-and-assignment v0.1.0
+
+The API definition(s) are based on
+* Commonalities v0.6.0 (r3.3)
+* Identity and Consent Management v0.4.0 (r3.3)
+
+## qos-booking v0.1.0
+
+**qos-booking v0.1.0 is the first initial release of this new API.**
```

Comment:
```suggestion
**qos-booking v0.1.0 is the initial public release of this new API.**
```

Reply:
I would choose "first public release" (as "initial" is also used for the API version). But that's more a question of style.